### DISCUSSION TITLE
Imagemagick Convert

### DISCUSSION THREAD

**Discussion by kechkibet** [dlemstra/magick-wasm#76] [General]
Hi, am trying to run the followng imagemagick  convert command:

convert in.tiff -set colorspace Gray -separate -average out.gray.tiff

is it possible to do from js via the library?
Kindly assist.
Thanks.

**Comment by dlemstra** [comment]
This is not yet possible. The `-average` option is a bit strange because that calls `evaluate` on the `MagickImageCollection` and that method is not yet available in the api. I am going to add this today so you can use that in the next releases. The rest of the options are available.
